src/reducers/descriptions/trackerCapture.reducerDescriptions.js

```diff
@@ -74,16 +74,15 @@ import { currentSelectionsReducerDesc } from './currentSelections.reducerDescrip
 import { feedbackDesc } from './feedback.reducerDescription';
 import { activePageDesc } from '../../core_modules/capture-core/reducers/descriptions/activePage.reducerDescription';
 import { searchPageDesc } from '../../core_modules/capture-core/reducers/descriptions/searchPage.reducerDescription';
+import { enrollmentPageDesc } from '../../core_modules/capture-core/reducers/descriptions/enrollmentPage.reducerDescription';
 import { newPageDesc } from '../../core_modules/capture-core/reducers/descriptions/newPage.reducerDescription';
 
 export default [
```

Comment:
I ordered alphabetically here, since it was already there I thought is good if we have it complete. 
Sorry this is out of scope for this PR.